test/test_google_drive.rb

```diff
@@ -36,7 +36,7 @@ def test_spreadsheet_append
     assert { ss.title == ss_title }
     ws = ss.worksheets[0]
     
-    ss.append("A1", [ %w[abc def ghi], %w[jkl mno pqr] ])
+    ss.append_values("A1", [ %w[abc def ghi], %w[jkl mno pqr] ])
     ws.reload
     assert { ws.max_rows == 1002 }
     assert { ws.max_cols == 26 }
```

Comment:
I guess these values are not guaranteed (depending on the default size of the worksheet). Can you remove these assertions?